src/shims/unix/linux/epoll.rs

```diff
@@ -51,6 +51,8 @@ impl EpollEventInstance {
 #[derive(Clone, Debug)]
 pub struct EpollEventInterest {
     /// The file descriptor value of the file description registered.
+    /// This is only used as the key of ready_list, so the exact value is stored
+    /// instead of reference.
```

Comment:
```suggestion
    /// This is only used for ready_list, to inform userspace which FD triggered an event.
    /// For that, it is crucial to preserve the original FD number.
    /// This FD number must never be "dereferenced" to a file description inside Miri.
```
I think this is right?

Reply:
Yup.